### ISSUE TITLE
Should the training set and test set unify the center of mass?

### ISSUE THREAD

**Issue by ting9059** [intel-isl/Open3D-PointNet2-Semantic3D#48]
Thank you for releasing this proposal.I want to use this code to run my own data set, and I find that the center of each scene in the semantic3D dataset and scannet dataset is very close, but the center of mass of each scene in my own dataset is quite different, may I ask whether this will affect the training results?

**Comment by Tojens** [comment]
I have the same question. Did you figure anything out?
